src/components/RecentList/styles.scss

```
@@ -0,0 +1,44 @@
+@import '../../../assets/scss/variables/index.scss';
+@import '../../../assets/scss/mixins/index.scss';
+.recent-list{
+    margin-top: 40px;
+    margin-left: $menuWidth;
+    @include padding( 0 50px);
+    max-width: calc(100vw - $menuWidth);
+    &__title{
```

thêm một dấu space trước { em nhé, nhìn nó đẹp với chuẩn chỉ hơn, những chỗ khác sửa tương tự nhé
```suggestion
&__title {
```